### PR TITLE
Remove obsolete platform services method

### PR DESCRIPTION
**Pull request by Strifex**
### :sparkles: What kind of change does this PR introduce? (Bug fix, feature, docs update...)

This PR removes the obsolete InitializePlatformServices method from MvxSetup's startup process.  It also updates the WpfSetup to utilize the proper InitializeFirstChance method instead to be consistent with all the other platforms.

### :arrow_heading_down: What is the current behavior?

Platforms can utilize the InitializePlatformServices method (marked obsolete) instead of InitializeFirstChance and the internals of the wpf setup class also utilize it.

### :new: What is the new behavior (if this is a feature change)?

Developers can no longer utilize the obsolete InitializePlatformServices method in MvxSetup and Wpf platform no longer relies on it.

### :boom: Does this PR introduce a breaking change?

Yes, if a developer overrode InitializePlatformServices in MvxSetup, even though it was obsolete - then they will have to change to InitializeFirstChance method instead.

### :bug: Recommendations for testing

Run the Wpf playground project and other mvx projects to see that no behavior changed.

### :memo: Links to relevant issues/docs

N/A

### :thinking: Checklist before submitting

- [X] All projects build
- [X] Follows style guide lines ([code style guide](https://github.com/MvvmCross/MvvmCross#code-style-guidelines))
- [X] Relevant documentation was updated ([docs style guide](https://www.mvvmcross.com/documentation/contributing/mvvmcross-docs-style-guide))
- [X] Rebased onto current develop

### CODE REVIEW

File: MvvmCross/Platforms/Wpf/Core/MvxWpfSetup.cs [hunk 1]
```diff
@@ -97,10 +91,14 @@ protected override IMvxNameMapping CreateViewToViewModelNaming()
...
 
+        protected override void InitializeFirstChance()
+        {
+            RegisterPresenter();
+        }
+
         protected override void InitializeLastChance()
         {
             InitializeBindingBuilder();
-            base.InitializeLastChance();
```
Comment (nickrandolph) [inline MvvmCross/Platforms/Wpf/Core/MvxWpfSetup.cs]: Why are we removing this call? It's good practice to call the base method, even though currently the base method doesn't do anything.

File: MvvmCross/Platforms/Wpf/Core/MvxWpfSetup.cs [hunk 2]
```diff
@@ -97,10 +91,14 @@ protected override IMvxNameMapping CreateViewToViewModelNaming()
             return new MvxPostfixAwareViewToViewModelNameMapping("View", "Control");
         }
 
+        protected override void InitializeFirstChance()
+        {
```
Comment (nickrandolph) [inline MvvmCross/Platforms/Wpf/Core/MvxWpfSetup.cs]: See my other comment - we should call the base method ie base.InitializeFirstChance()

Comment (Strifex) [reply MvvmCross/Platforms/Wpf/Core/MvxWpfSetup.cs]: @nickrandolph my mistake - this was an oversight when I was removing some test code I must have removed the base calls as well - its fixed now.

File: MvvmCross/Platforms/Wpf/Core/MvxWpfSetup.cs (hunk 1 above)
Comment (Strifex) [reply MvvmCross/Platforms/Wpf/Core/MvxWpfSetup.cs]: @nickrandolph my mistake - this was an oversight when I was removing some test code I must have removed the base calls as well - its fixed now.

**Comment by Strifex** [comment]
@Cheesebaron looks like build bot failed here but if builds fine on two different machines I have - problem with build machine potentially?

Error in the pipeline looks like the following:

C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\MSBuild\Xamarin\Android\Xamarin.Android.Common.targets(1669,2): error XA0000: Unhandled exception: System.IO.IOException: The process cannot access the file 'D:\a\1\s\MvvmCross.Plugins\Location.Fused\obj\Release\monoandroid90\android\src\com\google\android\gms\common\R.java' because it is being used by another process.

**Comment by Cheesebaron** [comment]
@Strifex should be fixed now if you either rebase on develop or merge develop into your branch.

**Comment by Strifex** [comment]
@Cheesebaron updated!

**Pull request merged**